hedera-node/test-clients/src/main/resource/contract/contracts/OpcodesContract/OpcodesContract.yul

```diff
@@ -0,0 +1,347 @@
+/// @use-src 0:"OpcodesContract.sol"
+object "NewOpcodes_51" {
```

Comment:
I think this is the first `*.yul` file added right?
Going forward what's the suggestion do you now have to add a `.sol` & a `.yul`, also how should it be utilized?

Reply:
This is because Solidity doen't support PUSH0 yet in assembly or YUL.  Building instructions are at the top of the .sol file.

Reply:
Perhaps add that first sentence as an additional comment up at the top of this file with the building instructions?  It's kind of a `TODO` isn't it?  When Solidity gets versioned we can take it and fix this?